internal/kafka/internal/workers/kafka_mgrs/accepted_kafkas_mgr.go

```diff
@@ -133,6 +134,7 @@ func (k *AcceptedKafkaManager) markTheUnassignedKafkaAsFailedOrAllowRetryCluster
 		return errors.Wrapf(err2, "failed to update failed kafka %s", kafka.ID)
 	}
 	metrics.UpdateKafkaRequestsStatusSinceCreatedMetric(constants2.KafkaRequestStatusFailed, kafka.ID, kafka.ClusterID, durationSinceCreation)
+	metrics.UpdateKafkaRequestsStatusSinceLastSetMetric(constants2.KafkaRequestStatusFailed, kafka.ID, kafka.ClusterID)
```

Comment:
let's only update this if the status is not already `Failed`. This is called on every reconcile so it will get updated every few seconds.